extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -31,7 +31,7 @@ import {tryParseJson} from '../../../src/json';
 /** @const */
 const TAG = 'amp-subscriptions';
 
-/** @typedef {{loggedIn: boolean, subscribed: boolean, granted: boolean, entitlement: !JsonObject}} */
+/** @typedef {{loggedIn: boolean, subscribed: boolean, granted: boolean, entitlement: !JsonObject, metering: ./entitlement.MeteringData}} */
```

Comment:
Let's replace `metering:Array` to `metered:boolean` more in keeping with this structure.

Reply:
Done